src/bandersnatch/tests/test_mirror.py

```diff
@@ -1063,3 +1064,7 @@ def test_determine_packages_to_sync(mirror: BandersnatchMirror) -> None:
     mirror.packages_to_sync = {"black": 69, "foobar": 47, "barfoo": 68}
     target_serial = mirror.find_target_serial()
     assert target_serial == 69
+
+
+if __name__ == "__main__":
+    pytest.main(sys.argv)
```

Comment:
Was this just so you could run it directly to test? Can also do that with `pytest` and some args I always forget.

Reply:
Yes, and just making it consistent with test_verify.py. I suppose all test files should have it present or not consistently.